README.md

```diff
@@ -14,3 +14,9 @@ Score management made easy
 ## Contribution
 
 Commit message: prefix with an [meaningful emoji](https://gitmoji.dev)
+
+## Mocking Next Auth during Development
+
+To mock NextAuth in development, you should set the environment variable `NEXT_PUBLIC_MOCK_ROLE` to `admin`, `ta` or `student` in your `.env` file.
+
+⚠️ Switching between mocking or not needs manual refresh on the client side. ⚠️
```

Comment:
The needs of refreshing manually is caused by caching from @tanstack/react-query (used by trpc).

Reply:
good